packages/headless/src/components/menu/index.ts

```diff
@@ -0,0 +1 @@
+export * from './menu';
```

Comment:
In this case, only one file in the folder, we used to export the component directly form the specific file and not from index